src/Web/Telegram/API/Bot/Data.hs

```diff
@@ -105,8 +106,13 @@ instance FromJSON Contact where
 
 -- | This object represents a chat.
 data Chat = Chat
-  {
-    chat_id :: Int                -- ^ Unique identifier for this chat, not exceeding 1e13 by absolute value
+  { chat_id :: Int64
```

Comment:
I believe that both chat ids should have the same maximum value. It is the same chat id, isn't it? so I would change it to `Int64`.